### PR TITLE
Add missing wall tiles to the interior tileset

### PR DESCRIPTION
**Pull request by abcdefg30**
See https://github.com/OpenRA/OpenRA/issues/13035#issuecomment-290287558.

### CODE REVIEW

**Review by ltem — APPROVED** [review]

File: mods/ra/tilesets/interior.yaml [hunk 1]
```diff
@@ -1181,3 +1181,192 @@ Templates:
 		Categories: Floor
 		Tiles:
 			0: Clear
+	Template@291:
+		Id: 291
```
Comment (pchote) [inline mods/ra/tilesets/interior.yaml]: Are these the correct template IDs for these tiles?

Try importing the "Mousetrap" aftermath mission to confirm?

Comment (abcdefg30) [inline mods/ra/tilesets/interior.yaml]: They should be. I used `Sarin Gas 2: Down Under` as testcase (`Mousetrap` doesn't have those walls?).

**Review by Mailaender — DISMISSED** [review]
`lwal001.int` to `lwal004.int` has 3 variants. `lwal005.int` has 6. You can set them up with additional `Tiles:` and `PickAny`. See `gstr0002` for reference.

File: mods/ra/tilesets/interior.yaml [hunk 2]
```diff
@@ -1181,3 +1181,214 @@ Templates:
...
+		Id: 292
+		Images: lwal0002.int
+		Size: 1,1
+		PickAny: True
+		Categories: Wall
+		Tiles:
+			0: Wall
+			1: Clear
+			2: Clear
+			3: Clear
```
Comment (reaperrr) [inline mods/ra/tilesets/interior.yaml]: This in combination with PickAny means there's a 25% chance the map editor will chose an impassable tile. Is that really intended?

Comment (pchote) [reply mods/ra/tilesets/interior.yaml]: This is almost certainly wrong - there is no meaningful difference between the four artwork frames.

These are walls, so why are *any* of them marked as Clear?

Comment (abcdefg30) [reply mods/ra/tilesets/interior.yaml]: -.- copy paste error. Sorry.

**Review by Mailaender — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a96f371a4ec9ba84b5cb081559d872fdcfde0cd0...e735067e9a1e7b119e672f8aeec8a872225e9e62

**Pull request merged**